test/e2e/revision_test.go

```diff
@@ -25,25 +25,25 @@ import (
 	"gotest.tools/assert"
 )
 
-func TestRevisionWorkflow(t *testing.T) {
+func TestRevision(t *testing.T) {
+	t.Parallel()
 	test := NewE2eTest(t)
 	test.Setup(t)
 	defer test.Teardown(t)
 
-	t.Run("create hello service and returns no error", func(t *testing.T) {
+	t.Run("create hello service and return no error", func(t *testing.T) {
```

Comment:
this is a small issue, however, I disagree with this change as I mentioned before. The title of the test is for humans, “returns” is the correct Eglish. Changing to “return” here has the feeling that the PR author is just looking for things to change.

Cc: @rhuss I guess this is already merged but perhaps we should have resolutions on various reviewers before merging

Reply:
Have you seen my comment https://github.com/knative/client/pull/307#issuecomment-515656277 ?

"return" is perfectly valid as the sentence is an imperative sentence. If you change this to a declarative sentence, then you should use "returns" but also "creates". Is this hard to understand ? I think we should start a vote whether we want to use the imperative or declarative form for *test* output (that no users see).

Reply:
> Cc: @rhuss I guess this is already merged but perhaps we should have resolutions on various reviewers before merging

Can we agree that the imperative form is ok ? It the same form that is also recommended for git commit message and it says: "Hey, test, create hello service and return no error"

You could also say "this test create**s** hello service and return**s** no error"

Regardless what, the sentence "create hello service and returns no error" is plain wrong.

Reply:
But you probably just overlooked https://github.com/knative/client/pull/307#issuecomment-515656277 because I haven't seen any answer or rejection of it. So I thought that you silently agreed (like for my other review comments elsewhere. maybe.)

Reply:
And as also stated before, I firmly believe we should not waste our energy here. there are much bigger challenges ahead.

Reply:
My intention was really just to fix the test name so as to use the imperative style of the sentence. I was not looking for random things to change, really.

Reply:
Did not see the comment. I am cool. I also really like the Parallel change which justifies this change but clearly came after.

We are cool @mgencur and certainly appreciate the contributions. Cheers 🍻

Reply:
In general, as you know from previous interactions, I try not to be picky about things. My goal is to ship things: the best UX for Knative. All of this is a means to an end.

That said, the more consistent the code and our work products the better. So just wanted to be clear. We are now. Enough words on this issue. Cheers all. Next :)